pages/programs/index.tsx

```diff
@@ -104,6 +107,8 @@ const pastPrograms: ProgramListingProps[] = [
 ];
 
 const Programs: React.FC = () => {
+  const createProgramModalManagement = useDisclosure();
```

Comment:
Did we forget to include the the CreateProgramModal on this page? I don't see it locally

![image](https://user-images.githubusercontent.com/71473137/199623375-7eb40bad-d122-45dd-a1b2-577419e4ff7e.png)

Reply:
Fixed that up